content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Use a SOCKS5 Proxy to Access the Kubernetes API
+content_type: task
+weight: 42
+min-kubernetes-server-version: v1.24
+---
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
+
+This page shows how to use a SOCKS5 proxy to access the API of a remote Kubernetes cluster.
+This is useful when the cluster you want to access does not expose its API directly on the public internet.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+You need SSH client software (the `ssh` tool), and an SSH server running on the remote server.
+You must be able to log in to the SSH server on the remote server.
+
+<!-- steps -->
+
+## Task context
+
+{{< note >}}
+This example tunnels traffic using SSH, with the SSH client and server acting as a SOCKS proxy.
+You can instead use any other kind of [SOCKS5](https://en.wikipedia.org/wiki/SOCKS#SOCKS5) proxies.
+{{</ note >}}
+
+Figure 1 represents what you're going to achieve in this task.
+
+* You have a client computer from where you're going to create requests to talk to the Kubernetes API
```

Comment:
Do we want to mention here that the client computer will be referred to as local in the diagram/steps ahead? I understand we explicitly speak about how localhost is not the local client computer somewhere below. But I think it'd help if we state this explicitly for step #3.

```suggestion
* You have a client computer, referred to as local in the steps ahead, from where you're going to create requests to talk to the Kubernetes API.
```